test/integration/integration_test.py

```diff
@@ -0,0 +1,75 @@
+from marquez_client.client import Client
+
+import logging
+import os
+import pytest
+import requests
+import subprocess
+import sys
+from urllib3.util.retry import Retry
+
+
+def test_data_in_marquez():
+
+    dag_id = "test_dag_v2"
+    execution_date = "2019-02-01T00:00:00"
+    namespace = "integration-test"
+
+    c = Client(namespace_name=namespace)
+    wait_for_marquez()
```

Comment:
wait_for_marquez should be a test fixture

Reply:
for now it's fine,  we have just one test.

Reply:
I think it's worth moving. The body of the test should be as clean as possible with only the 
when -> then type logic. Code that's involved in setup or teardown should aggressively be moved out of the body.

FWIW, it's just a 3-line change :) (add the fixture tag, add the fixture to the test constructor, and remove the function call from the body).

P.S. If you want to have one fixture called explicitly after another, you can add one fixture into the constructor of another to enforce that ordering.

Reply:
done

Reply:
Thanks, appreciate it!